hack/integration-tests-local.sh

```diff
@@ -9,32 +9,45 @@ export AWS_DEFAULT_REGION=us-east-1
 export AWS_ENDPOINT=localhost:9000
 export AWS_DISABLE_SSL=true
 
-docker container run -d --name helm-s3-minio \
+DOCKER_NAME='helm-s3-minio'
+
+cleanup() {
+  if $(docker container ls | grep -q "${DOCKER_NAME}\$") ; then
```

Comment:
We should omit `$()` here:

```suggestion
  if docker container ls | grep -q "${DOCKER_NAME}$" ; then
```

Reply:
fixed